backend/dal/Services/Concrete/PropertyService.cs

```diff
@@ -147,9 +147,9 @@ public IEnumerable<PropertyModel> Search(AllPropertyFilter filter)
             var projectNumbers = properties.SelectMany(p => JsonSerializer.Deserialize<IEnumerable<string>>(p.ProjectNumbers ?? "[]")).Distinct().ToArray();
             var statuses = from p in this.Context.ProjectProperties
                            where projectNumbers.Contains(p.Project.ProjectNumber)
-                           select new { p.Project.ProjectNumber, p.Project.Status, WorkflowCode = p.Project.Workflow.Code };
+                           select new { p.Project.ProjectNumber, p.Project.Status, WorkflowCode = p.Project.Workflow.Code, p.Project.Status.IsTerminal };
 
-            foreach (var status in statuses)
+            foreach (var status in statuses.Where(s => !s.IsTerminal))
```

Comment:
same here, do not return project metadata for projects in terminal statuses, otherwise markers will incorrectly be shown as red/purple.